.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,36 @@
+name: "Feature request ✨"
+description: "Create a feature request to help Ragna improve"
+title: "[ENH] - <title>"
+labels: ["type: enhancement 💅"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for using Ragna and taking the time to contribute to this project.
+  - type: textarea
+    attributes:
+      label: Feature description
+      description: |
+        Describe what you are proposing. Provide as much context as possible and link to related issues and/or pull requests.
+        This section should contain "what" you are proposing.
+        Are you having any problems? Briefly describe what your painpoints are. For example: "I'm always frustrated when ..."
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Value and/or benefit
+      description: |
+        What is the value of adding this feature, and who will benefit from it?
+        Include any information that could help us prioritize the issue.
```

Comment:
I'm always frustrating to see these two separated from each other. If I actually write something like "I'm always frustrated when ..." in the first, the second from just becomes "The benefit is that I'm no longer frustrated".

What value do you think we'll have by having two fields here?

Reply:
It's separated to encourage contributors to think about this, and manage expectations if their request is not prioritized. While seasoned contributors include this information by default, there are many people who don't.

For example: If the only benefit is that it's slightly less frustrating to one user, it may not get worked on soon - but if they share that the feature will enable their team/project/community to adopt Ragna, we can try to prioritize it.

If it still feels unnecessary, we can remove it.

Reply:
(Aside, we can simplify the description for the first prompt. Will update in next commit.)